rllib/algorithms/ppo/tests/test_ppo_rl_module.py

```diff
@@ -177,18 +188,26 @@ def test_forward_train(self):
                 # CNN not implement in TF2 yet
                 continue
             print(f"[FW={fw} | [ENV={env_name}] | LSTM={lstm}")
-            env = gym.make(env_name)
+            # TODO(Artur): Figure out why this is needed and fix it.
+            if self.env.startswith("ALE/"):
+                env = gym.make("GymV26Environment-v0", env_id=self.env)
+            else:
+                env = gym.make(env_name)
 
+            preprocessor = get_preprocessor(env.observation_space)(
```

Comment:
why do we construct and transform on the same input?

Reply:
We don't transform. `get_preprocessor` returns a type, which takes the observation space as an input (again).

Reply:
ok got it.Let's do it in two parts to be more clear.
```
preprocessor_cls = get_preprocessor(env.observation_space)
preprocessor = prerprocessor_cls(env.observation_space)
```

Reply:
Done